server/utils/sendSlackMessage.js

```diff
@@ -0,0 +1,11 @@
+const axios = require('axios');
+
+const sendSlackMessage = (slackID, customMessage) => {
+  const slackOAuthToken = process.env.SLACK_OAUTH;
+
+  const slackPostMessageURL = `https://slack.com/api/chat.postMessage?token=${slackOAuthToken}&channel=${slackID}&text=${customMessage}`;
+
+  axios.post(slackPostMessageURL);
```

Comment:
Just a check as I don't know the answer either but this POST request looks unusually bare (no body) so are we sure this should be a POST request and not a GET?

Reply:
its definitely a post request, the only info sent is through params
https://api.slack.com/methods/chat.postMessage

Reply:
Understood. I'm wondering whether that information, currently sent through the params, can be sent through in the body then? I don't think it's good practice to send info through params as part of a POST request.